### ISSUE TITLE
arc-green as default

### ISSUE THREAD

**Issue by ShalokShalom** [go-gitea/gitea#10489]
Gitea.com is how the people see Gitea the first time.
Did you consider a dark theme as the default?

This could help to differentiate yourself from Github and Discord got very successful with it.

Hopefully, this gets chosen as default, since it is healthy for our eyes, who suffer from the blue light spectrum in the bright background.

**Comment by lafriks** [comment]
I would not mind, maybe add just add notification kind of switch to dark theme?

**Comment by ShalokShalom** [comment]
A news article would be nice. Honestly, I think power users are more likely to switch to a dark theme, since this is what they usually do. Also the reason why VSCode is dark by default. Same goes for Atom.

**Comment by silverwind** [comment]
`arc-green` has many quality issues, it's not as polished as the light theme. I would not recommend it as default.

I'm working on a new dark theme in https://github.com/go-gitea/gitea/pull/10430. 

It may be possible to automatically set default light/dark via querying [@prefers-color-schema](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme) in JS.

**Comment by lunny** [comment]
I personally still like the current theme. :)

**Comment by septatrix** [comment]
Honestly I would prefer to implement an automatic mode as proposed in #10927 and mentioned by @silverwind in his comment above.

**Comment by septatrix** [comment]
Blocked by first needing a more accessible dark theme and may be superseded by #10927
